src/getMetroConfig.js

```diff
@@ -46,33 +46,48 @@ module.exports = symlinkedDependencies => {
 
     const extraNodeModules = peerDependenciesOfSymlinkedDependencies
         .map(mapModule)
-        .join(',\n')
+        .join(',\n  ')
 
-    const getBlacklistRE = symlinkedDependenciesPaths.map(mapPath).join(',\n')
+    const getBlacklistRE = symlinkedDependenciesPaths
+        .map(mapPath)
+        .join(',\n  ')
 
     const getProjectRoots = symlinkedDependenciesPaths
-        .map(path => `    path.resolve('${path}')`)
-        .join(',\n')
+        .map(path => `path.resolve('${path}')`)
+        .join(',\n  ')
 
     return dedent`
       const path = require('path');
-      const blacklist = require('metro/src/blacklist');
 
-      module.exports = {
-        extraNodeModules: {
-      ${extraNodeModules}
-        },
-        getBlacklistRE: () =>
-          blacklist([
-      ${getBlacklistRE}
-          ]),
-        getProjectRoots: () => [
-          // Include current package as project root
-          path.resolve(__dirname),
-          // Include symlinked packages as project roots
-      ${getProjectRoots}
-        ],
```

Comment:
Regarding the spacing changes, notice how the indentation goes back and forth above. That goes away by moving the extra spaces to the `.join()` statements -- only additional lines will get the extra spaces, so we can properly format the first here.